src/styles/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin rotate-animation($duration, $timing-function) {
+  animation: hand-move $duration $timing-function infinite;
+  transform-origin: bottom center;
+
+  @keyframes hand-move {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
The '@keyframes' definition inside the mixin could lead to duplicated animations being generated if the mixin is included multiple times. Consider moving the '@keyframes' definition outside of the mixin to avoid duplication.